config/centos/default/grr-http-server

```diff
@@ -1,5 +0,0 @@
-# Configuration file for grr-http-server start-up script
```

Comment:
So what's happening with these centos files? I think you removed them to use the ones from the os-agnostic upstart directory. But I don't see any corresponding build code change for rpms (only debs). How does the rpmbuilder know where to find them now?